app/models/export/submissions/extract.rb

```diff
@@ -0,0 +1,24 @@
+module Export
+  class Submissions
+    module Extract
+      RELEVANT_STATUSES = %w[completed validation_failed].freeze
+
+      def self.all_relevant
+        Submission.select('submissions.*,' \
```

Comment:
👏🏽 on getting this done in a single query. what's your feeling on how this will perform as the number of submissions grow?